examples/Cargo.toml

```
@@ -46,7 +46,11 @@ members = [
 ]
 
 exclude = [
-    "binaries/*"
+    "binaries/*",
+    "binaries/new-meta/io",
+    "binaries/new-meta/state-v1",
+    "binaries/new-meta/state-v2",
+    "binaries/new-meta/state-v3",
```

```suggestion
    "binaries/**",
```
[ref](https://docs.rs/glob/0.3.0/glob/struct.Pattern.html)

---

I've tried `binaries/**`, `binaries/**/*`, `binaries/new-meta/*`, `binaries/new-meta/**` etc. but nothing worked to me.